tests/e2e/support/objects/app-files/share/utils.ts

```diff
@@ -0,0 +1,29 @@
+import { errors, Page } from 'playwright'
+import util from 'util'
+
+const acceptedShareList =
```

Comment:
IMO, it is selecting a single row so item would be nice
```suggestion
const acceptedShareItem =
```